.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,42 @@
+name: Integration Tests
+
+on: [pull_request, push]
+
+jobs:
+  fprime-integration:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+        fprime-version: [[NASA-v1.5.3, 99cef07], [v2.0.0, 521516c] , [devel, 521516c]]
```

Comment:
That's really cool! @LeStarch would you be okay with tagging the gpsapp so we don't have to specify random commit hashes here? Also could you add a commit specifying that this tuple is [fprime version, gpsapp version] so that there's no confusion.

Reply:
Now that we're just using the fprime version that comes with the gps-tutorial version, how do we want to layout the tests? The fprime versions and the gps-tutorial versions aren't necessarily linked anymore so do we still want to test the older gps-tutorial?

Reply:
@Joshua-Anderson tag away!